packages/@o3r/extractors/src/utils/localization.ts

```diff
@@ -18,7 +18,7 @@ export function getLocalizationFileFromAngularElement(node: ts.ClassDeclaration)
         !ts.isDecorator(decorator)
         || !ts.isCallExpression(decorator.expression)
         || !ts.isIdentifier(decorator.expression.expression)
-        || decorator.expression.expression.escapedText !== localizationDecoratorName
+        || decorator.expression.expression.escapedText.toString() !== localizationDecoratorName
```

Comment:
I thought the rule was preventing us from using `.toString()` (@typescript-eslint/no-base-to-string), is it ok here?

Reply:
Here it's okay as `escapedText` expose explicitly a `toString` method